examples/sv/sv.rat

```diff
@@ -0,0 +1,22 @@
+# This is an attempt to rewrite the model here: https://github.com/pourzanj/homebuying/blob/master/stan/sv.stan
+y ~ normal(mu_ret + rho * v[t] * s[t], s[t]);
+
+s[t] = exp((mu + h[t]) / 2.0);
+
+# Here's our recursion!
+h[t] = phi * h[shift(t, 1)] + v[t];
+h[1] = v[1] / sqrt(1 - phi * phi);
```

Comment:
So for this line, do we consider `h[1]` to denote the first element value of the subscript set `t`? Do we assume some ordering exists for the subscript set and `h[1]` denotes the first element of the index set of the subscript?